### ISSUE TITLE
New release for Nextcloud 21?

### ISSUE THREAD

**Issue by e-alfred** [nextcloud/issuetemplate#326]
Hello,

is it possible to make a new release for Nextcloud 21?

**Comment by enoch85** [comment]
And 22...? :)

**Comment by enoch85** [comment]
This script will help anyone that wants to enable it:

```
#!/bin/bash

source /var/scripts/fetch_lib.sh
nc_update

rm -R "$NCPATH"apps/issuetemplate
curl_to_dir https://github.com/nextcloud/issuetemplate/releases/download/v0.7.0 issuetemplate.tar.gz /tmp
tar -xvf /tmp/issuetemplate.tar.gz -C "$NCPATH"/apps
bash "$SECURE"
sed -i "s|20|${CURRENTVERSION%%.*}|g" "$NCPATH"/apps/issuetemplate/appinfo/info.xml
nextcloud_occ app:enable issuetemplate
```

On your own risk.

**Comment by szaimen** [comment]
I think better is this:
https://github.com/nextcloud/vm/pull/2099#issuecomment-908724096

**Comment by iGadget** [comment]
> And 22...? :)
...and what about 23? :-S

**Comment by TP75** [comment]
Happy New Year and a kind reminder.

@juliushaertl Can you push a new release please? Nextcloud 21 until 23 is out now for some time...

**Comment by Dubidubiduu** [comment]
White screen in 24 admin setting

**Comment by enoch85** [comment]
Now we want 24! :D

**Comment by p-bo** [comment]
Just in case it wasn't noticed - Nextcloud improved policy regarding changes in core to ease app developers in maintaining compatibility with newer Nextcloud versions:

https://help.nextcloud.com/t/new-process-for-documenting-core-code-changes-that-affect-app-developers/149828?u=p-bo

**Comment by TP75** [comment]
Hav a loot at https://github.com/nextcloud/issuetemplate/issues/415

**Comment by Mannshoch** [comment]
We need a release for 27

**Comment by SimJoSt** [comment]
Official Nextcloud apps, like Nextcloud Talk (Spreed), still suggest using the output of this app to fill out their bug report issue templates: https://github.com/nextcloud/spreed/blob/v22.0.7/.github/ISSUE_TEMPLATE/1_bug_report.md?plain=1#L65
Without a running version on current versions of Nextcloud, this feels like a little disorganized :)

**Comment by SimJoSt** [comment]
I've updated the title of the issue to the currently maintained Nextcloud versions.

On the [app store page](https://apps.nextcloud.com/apps/issuetemplate) there is a label/badge showing
> Orphan / Looking for Maintainer
<img width="890" height="244" alt="Image" src="https://github.com/user-attachments/assets/83017dd6-0140-42cf-9d9d-0c93726aee54" />

The screenshot is also not loading any more, same as in the README.md in this repo.

However, on May 28, 2023 the supported version were raised to 24-28 (https://github.com/nextcloud/issuetemplate/commit/9d149f874073002c6237c711595d4efa030422f9) even though this change was never released in a new version.

@juliusknorr @nickvergessen is this app officially orphaned and are you looking for contributors, like the notice on the app store page?
If yes, it would be great to have this notice also communicated in this issue and on the repositories main page.

If not, what need to happen to make it work with currently maintained version of Nextcloud?
The app is really helpful for non-technical people to create issues with helpful context.

@

**Comment by nickvergessen** [comment]
Well by now the issue template on github is good enough, so I wouldn't invest more into this.
If someone takes it over fine, otherwise we can simply archive it

**Comment by SimJoSt** [comment]
Thank you for the reply 🙏

Are you saying that the issue templates in the repositories provide sufficient instructions, including example commands for `occ`, to help users access the data they need?
I always thought, the idea of the app was to centralize the template creation, so not every repository needs to replicate it and stay up to date. This could also be interesting to app developers that are not part of the Nextcloud organisation. They benefit from clean issues as well.

As of the latest change from last week, the bug report template from Nextcloud Talk still references this app: https://github.com/nextcloud/spreed/blob/d44114764e88300495f37e9e863e4f4e728270ab/.github/ISSUE_TEMPLATE/1_bug_report.md?plain=1#L66

I personally would prefer to have the app work again, as it saves me a lot of time when creating issues, not having to run `occ` commands, etc.
However, I understand your position as well.

If somebody would be interested in maintaining the app, what should they do to get going?

**Comment by nickvergessen** [comment]
> I always thought, the idea of the app was to centralize the template creation, so not every repository needs to replicate it and stay up to date

yeah by now GitHub allows to have a template on organization level: https://github.com/nextcloud/.github/blob/master/.github/ISSUE_TEMPLATE/1_bug_report.md
It is used when the app doesnt have a custom one.

> As of the latest change from last week, the bug report template from Nextcloud Talk still references this app

Never change a running system.

> what should they do to get going?

Fix the app so it works again, afterwards request a new certificate in https://github.com/nextcloud/app-certificate-requests

**Comment by SimJoSt** [comment]
Nice, I didn't know organization level issue templates. That can solve it for the repositories under the Nextcloud organization banner.

> Never change a running system.

Well, the system / this app, is in fact not running ;)

> Fix the app so it works again

By providing a PR? Most people won't have direct write access to this repository.
Or is there a specific way of onboarding somebody to become the maintainer with the relevant permissions?

**Comment by nickvergessen** [comment]
> By providing a PR? Most people won't have direct write access to this repository.

Should still always send a PR, so changes are better visible and documented
